tests/test_cache.py

```diff
@@ -116,6 +115,3 @@ def test_reverse_dependencies(from_line, tmpdir):
         "bottom": {"middle", "top"},
         "bonus": {"top"},
     }
-
-    # Clean up our temp directory
-    rmtree(tmpdir)
```

Comment:
pytest deletes tempdirs already. The extra delete here causes one test to fail after making all tests default to use empty pip.conf. The delete here causes the other fixture to fail [here](https://github.com/jazzband/pip-tools/blob/73a8a6e4c91e2f9a90642ed46294fbe94b2cf419/tests/conftest.py#L208).